src/libsync/syncfilestatustracker.cpp

```diff
@@ -25,20 +25,49 @@ namespace OCC {
 
 Q_LOGGING_CATEGORY(lcStatusTracker, "sync.statustracker", QtInfoMsg)
 
-static SyncFileStatus::SyncFileStatusTag lookupProblem(const QString &pathToMatch, const std::map<QString, SyncFileStatus::SyncFileStatusTag> &problemMap)
+static int pathCompare( const QString& lhs, const QString& rhs )
+{
+    // Should match Utility::fsCasePreserving, we want don't want to pay for the runtime check on every comparison.
```

Comment:
And yet, it does not becuase it does not check the env variable.
I suggest you still use that, is the runtime cost so high? 
If you really want to save the runtime cost on every call, i suggest you pass the case sensitivity as a parametter of the comparator.

Reply:
Arf just saw "approved" in the email and didn't see the comment.
The env variable is just for auto tests anyway, no?